MapboxVisionTests/RecordCoordinatorTests.swift

```diff
@@ -72,6 +70,69 @@ final class RecordCoordinatorTests: XCTestCase {
         XCTAssert(directoryExists(at: recordingPath), "Recording should be saved at \(recordingPath) after recording is stopped")
     }
 
+    class TestClipRequest {
+        let start: Float
+        let end: Float
+
+        init(start: Float, end: Float) {
+            self.start = start
+            self.end = end
+        }
+    }
+
+    func testVideoClipping(withRequests requests: [TestClipRequest], _ message: String = "", line: UInt = #line) {
+        let expectation = XCTestExpectation(description: "Wait for a big enough amount of time for record processing queue to finish its job")
```

Comment:
```suggestion
        let expectation = XCTestExpectation(description: "All the requested clips are successfully cut")
```
Is it what we really expecting?

Reply:
done